config/locales/idv/en.yml

```diff
@@ -39,16 +39,16 @@ en:
         Please check that the personal and financial information you provided is accurate.
         To learn more about the information we need from you and how it’s used to verify your identity,
         please see our troubleshooting page.
-      activated: >
+      activated_html: >
         Your identity has been verified. If you need to change your verified information,
-        please %{link}.
+        please <a href=%{url}>%{link_text}</a>.
```

Comment:
I would be fine with either approach, as long as we are doing it consistently 📈 . I do like the idea of using the `html` suffix and passing in the URL with a rails `link_to` helper. Seems clean.